src/Autofac.Integration.SignalR/RegistrationExtensions.cs

```diff
@@ -48,5 +50,18 @@ public static class RegistrationExtensions
                 .Where(t => typeof(IHub).IsAssignableFrom(t))
                 .ExternallyOwned();
         }
+
+        /// <summary>
+        /// Register types that implement <see cref="PersistentConnection"/> in the provided assemblies.
+        /// </summary>
+        /// <param name="builder">The container builder.</param>
+        /// <param name="controllerAssemblies">Assemblies to scan for controllers.</param>
+        /// <returns>Registration builder allowing the controller components to be customised.</returns>
+        public static IRegistrationBuilder<object, ScanningActivatorData, DynamicRegistrationStyle> RegisterConnections(this ContainerBuilder builder, params Assembly[] controllerAssemblies)

```

Comment:
"controllerAssemblies" should be renamed to "persistentConnectionAssemblies" for consistency